content/content-strategy/manage-content-requests/sign-off-content/index.yml

```diff
@@ -0,0 +1,17 @@
+pagetitle: Sign off content
+description: Create a brief or checklist for final sign off, so the approver knows what they need to do.
+weight: 70
+theme: blue
+header:
+  - /_shared/globalheader.md
+  - /_shared/header.md
+main:
```

Comment:
Missing the `why.md` file, and possibly its related heading partial. Add it where it needs to be in this list, or remove the `why.md` file from this folder.

Reply:
Trisha.done